key/aziot-keys/src/lib.rs

```diff
@@ -390,36 +608,57 @@ pub extern "C" fn cbindgen_unused_AZIOT_KEYS_SIGN_DERIVED_PARAMETERS(
     unimplemented!();
```

Comment:
(unrelated to this particular PR)

What's the point of this function?

Reply:
https://github.com/Azure/iot-identity-service/blob/408ed4afaf5bb866ab67fa8a87f845f4575984e9/key/aziot-keys/src/lib.rs#L49-L71